clients/web/src/rest.js

```diff
@@ -5,28 +5,75 @@ import Backbone from 'backbone';
 import events from 'girder/events';
 import { getCurrentToken, cookie } from 'girder/auth';
 
-var apiRoot = $('#g-global-info-apiroot').text().replace('%HOST%', window.location.origin) || '/api/v1';
-var staticRoot = $('#g-global-info-staticroot').text().replace('%HOST%', window.location.origin) || '/static';
+let apiRoot;
+let staticRoot;
 var uploadHandlers = {};
 var uploadChunkSize = 1024 * 1024 * 64; // 64MB
 
 /**
+ * Get the root path to the API.
+ *
+ * This may be an absolute path, or a path relative to the application root. It will never include
+ * a trailing slash.
+ *
+ * @returns {string}
+ */
+function getApiRoot() {
+    return apiRoot;
+}
+
+/**
  * Set the root path to the API.
  *
- * @param root The full root path for the API.
+ * @param {string} root The root path for the API.
  */
 function setApiRoot(root) {
-    apiRoot = root;
+    // Strip trailing slash
+    apiRoot = root.replace(/\/$/, '');
+}
+
+/**
+ * Get the root path to the static content.
+ *
+ * This may be an absolute path, or a path relative to the application root. It will never include
+ * a trailing slash.
+ *
+ * @returns {string}
+ */
+function getStaticRoot() {
+    return staticRoot;
 }
 
-/** Set the root path to the static content.
+/**
+ * Set the root path to the static content.
  *
- * @param root The full root path for the static content.
+ * @param {string} root The root path for the static content.
  */
 function setStaticRoot(root) {
-    staticRoot = root;
+    // Strip trailing slash
+    staticRoot = root.replace(/\/$/, '');
+    // publicPath would normally be set in the Webpack config file, but is not known at compile
+    // time.
+    __webpack_public_path__ = `${getStaticRoot()}/built/`; // eslint-disable-line no-undef, camelcase
```

Comment:
This feels wonky, like we shouldn't be having to do this. We use assets from CSS files in the item_tasks plugin, is the limitation just that we can't use them from pug files? Would it make more sense to set the `src` attribute programmatically from javascript at render time?